packages/components/PartyInfo.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Text } from '@chakra-ui/react';
+
+interface IPartyInfo {
```

Comment:
Minor

I beliave you could move this interface to our `/entities` folder in a new file, problably we could call the file election.
So, create a new file `/entities/election` and move this inteface there, so we keep our interfaces organized!

Reply:
Acredito que neste caso, como a interface não será re utilizada podemos mante-la aqui.